linux-loader/src/main.rs

```diff
@@ -122,5 +148,13 @@ mod tests {
     #[async_std::test]
     async fn test_pipe() {
         test("/bin/testpipe1").await;
+        let string = fs::read_to_string("../rootfs/testpipe.txt").unwrap();
+        assert_eq!(string, String::from("hello pipe\n"));
+        test("/bin/busybox rm testpipe.txt").await;
+    }
```

Comment:
I think the validation procedure should be moved into the test program, then it indicates test result (pass or failed) through the return value (0 or not).

Reply:
Thank you， I will fix these. ( It should be like this...